pixelpact-hooks/playwright-js/index.js

```diff
@@ -0,0 +1,123 @@
+import { readFileSync, existsSync, mkdirSync } from "fs";
+import fs from "fs/promises";
+import { dirname, resolve } from "path";
+
+const appDir = process.env.PWD;
+const config = JSON.parse(
+  readFileSync(`${appDir}/pixelpact.config.json`, {
+    encoding: "utf8",
+    flag: "r",
+  })
+);
+const folderPath = getFolderPath();
+
+const expectedFileSuffix = "expected";
+const actualFileSuffix = "actual";
+const diffFileSuffix = "diff";
+
+export async function toMatchVisually(page, testInfo, fileNamePrefix) {
```

Comment:
I think we can get rid of the `testInfo` (maybe in another PR) as follows:

Register a [`test.beforeEach`](https://playwright.dev/docs/api/class-test#test-before-each) hook, that stores the reference to `testInfo` somewhere (eg. a global in the module or maybe even AsyncLocalStorage). The  `toMatchVisually` can then obtain the `testInfo` instance from there, hiding this implementation detail from our public API.

We could also derive the `fileNamePrefix` from the `testInfo.title` if it is not specified.